arithmetization/src/main/java/net/consensys/linea/blockcapture/BlockCapturer.java

```diff
@@ -39,6 +38,10 @@
 import org.hyperledger.besu.plugin.data.BlockHeader;
 
 public class BlockCapturer implements ConflationAwareOperationTracer {
+  public static final int MAX_RELATIVE_BLOCK = 256;
```

Comment:
you could use `BLOCKHASH_MAX_HISTORY` in `GlobalConstants.java`

Reply:
Well maybe at some point I'll put that in